### PR TITLE
APPSREPO-652-Clean up Sync Service chart

### PR DESCRIPTION
**Pull request by ciprianchelaru**
Sync Service has it's own helm deployment project based on this one. So I removed any sync service reference from this project.

### CODE REVIEW

File: charts/incubator/alfresco-dbp/values.yaml [hunk 1]
```diff
@@ -204,7 +204,5 @@ alfresco-content-services:
     APP_CONFIG_OAUTH2_REDIRECT_LOGIN: "/workspace/"
     APP_CONFIG_OAUTH2_REDIRECT_LOGOUT: "/workspace/logout"
 
-alfresco-sync-service:
-  enabled: false
   alfresco-infrastructure:
```
Comment (svidrascu) [inline charts/incubator/alfresco-dbp/values.yaml]: These two properties belong to syn service also. please remove them.

Comment (svidrascu) [reply charts/incubator/alfresco-dbp/values.yaml]: https://github.com/Alfresco/alfresco-dbp-deployment/blob/845eac28278713a9710756caaa6f374f3be98858/charts/incubator/alfresco-dbp/values.yaml#L207-L208

**Comment by ciprianchelaru** [comment]
Yes @rgauss, Sync Service will be part of the ACS by default. Now sync service has this project https://git.alfresco.com/alfresco-services/alfresco-dsync-services-deployment so it's not needed anymore that aync service helm chart to be a part of the alfresco-dbp-deployment project.

**Comment by rgauss** [comment]
> Yes @rgauss, Sync Service will be part of the ACS by default. Now sync service has this project https://git.alfresco.com/alfresco-services/alfresco-dsync-services-deployment so it's not needed anymore that aync service helm chart to be a part of the alfresco-dbp-deployment project.

@ciprianchelaru, it doesn't looks like that's happened yet, so we can't remove it as a requirement from ours, and we'll need the sync AMPs applied in our repo image until the ACS repo image contains it as a default.

I'll comment specifically on what we'll need changed here.

File: charts/incubator/alfresco-dbp/requirements.yaml [hunk 2]
```diff
@@ -3,15 +3,11 @@ dependencies:
     version: 3.0.0
     condition: alfresco-content-services.enabled
     repository: https://kubernetes-charts.alfresco.com/incubator
-  - name: alfresco-sync-service
```
Comment (rgauss) [inline charts/incubator/alfresco-dbp/requirements.yaml]: We'll still need to pull this in until it's part of the ACS chart.

File: charts/incubator/alfresco-dbp/values.yaml [hunk 3]
```diff
@@ -175,8 +175,6 @@ alfresco-content-services:
       repository: quay.io/alfresco/alfresco-dbp-repository
       tag: "0.4-SNAPSHOT"
     environment:
-      # The SYNC_SERVICE_URI should contain the details of the ingress host and port.
-      SYNC_SERVICE_URI: "<ingresscontrollerurl>/syncservice"
```
Comment (rgauss) [inline charts/incubator/alfresco-dbp/values.yaml]: We'll still need to supply this value.

File: charts/incubator/alfresco-dbp/values.yaml [hunk 4]
```diff
@@ -185,7 +183,6 @@ alfresco-content-services:
         -Dalfresco.cluster.enabled=true
         -Ddeployment.method=HELM_CHART
         -Xms2000M -Xmx2000M
-        -Ddsync.service.uris=\"$SYNC_SERVICE_URI\"
```
Comment (rgauss) [inline charts/incubator/alfresco-dbp/values.yaml]: We'll still need to supply this value.

File: charts/incubator/alfresco-dbp/values.yaml [hunk 5]
```diff
@@ -203,8 +200,3 @@ alfresco-content-services:
     APP_CONFIG_OAUTH2_SILENT_LOGIN: "\"true\""
     APP_CONFIG_OAUTH2_REDIRECT_LOGIN: "/workspace/"
     APP_CONFIG_OAUTH2_REDIRECT_LOGOUT: "/workspace/logout"
-
-alfresco-sync-service:
```
Comment (rgauss) [inline charts/incubator/alfresco-dbp/values.yaml]: We'll still need this for now.

File: docker/repository/pom.xml [hunk 6]
```diff
@@ -10,18 +10,10 @@
 
     <properties>
         <alfresco.gs.repo.version>3.1.0</alfresco.gs.repo.version>
-        <alfresco.sync.service.version>3.1.0</alfresco.sync.service.version>
```
Comment (rgauss) [inline docker/repository/pom.xml]: We'll still need to add the Sync AMP until it's part of the default repo image.

**Comment by constantinpopa10** [comment]
Sync service is targeted to be included into ACS 6.1.1, both AMP and deployment.
There are a couple of pull requests on the ACS side waiting to be merged, but first we need to release a Sync service RC, which will happen very soon.
https://issues.alfresco.com/jira/browse/APPSREPO-693
https://issues.alfresco.com/jira/browse/APPSREPO-694

Let's keep this PR opened until our ACS PRs are merged.

**Comment by constantinpopa10** [comment]
The PRs mentioned above have been merged, both into master(chart version: 3.0.0) and support/SP/2.N (chart version 2.1.0)
This means the ACS deployment chart now contains Sync service.

I think this pull request can be merged now.

**Comment by rgauss** [comment]
It doesn't look like any of the discussions have been addressed.

File: charts/incubator/alfresco-dbp/requirements.yaml (hunk 2 above)
Comment (constantinpopa10) [reply charts/incubator/alfresco-dbp/requirements.yaml]: Sync service is part of the ACS chart now, for both charts of ACS 6.1.1 and ACS 6.2.0

File: charts/incubator/alfresco-dbp/values.yaml (hunk 3 above)
Comment (constantinpopa10) [reply charts/incubator/alfresco-dbp/values.yaml]: This property is defined in the ACS chart configuration, doesn't need to exist here anymore.

File: charts/incubator/alfresco-dbp/values.yaml (hunk 4 above)
Comment (constantinpopa10) [reply charts/incubator/alfresco-dbp/values.yaml]: See the comment above.

File: charts/incubator/alfresco-dbp/values.yaml (hunk 5 above)
Comment (constantinpopa10) [reply charts/incubator/alfresco-dbp/values.yaml]: Sync is included in the ACS chart at this very moment.

File: docker/repository/pom.xml (hunk 6 above)
Comment (constantinpopa10) [reply docker/repository/pom.xml]: Sync is part of the default repo image, both ACS 6.1.1 and latest(6.2.0)

**Comment by constantinpopa10** [comment]
Discussions have been addressed.

File: charts/incubator/alfresco-dbp/values.yaml (hunk 3 above)
Comment (rgauss) [reply charts/incubator/alfresco-dbp/values.yaml]: We still need to set the URI based on the domain being used for ingress into the cluster.

File: charts/incubator/alfresco-dbp/values.yaml (hunk 4 above)
Comment (rgauss) [reply charts/incubator/alfresco-dbp/values.yaml]: We still need to set the URI based on the domain being used for ingress into the cluster.

File: charts/incubator/alfresco-dbp/values.yaml (hunk 3 above)
Comment (constantinpopa10) [reply charts/incubator/alfresco-dbp/values.yaml]: Reverted the change.

File: charts/incubator/alfresco-dbp/values.yaml (hunk 4 above)
Comment (constantinpopa10) [reply charts/incubator/alfresco-dbp/values.yaml]: I understand, makes perfect sense. I reverted the change.

**Comment by constantinpopa10** [comment]
I reverted the changes that removed the property dsync.service.uris.

**Pull request merged**